### DISCUSSION TITLE
Intellij IDE Unable to login to github copilot

### DISCUSSION THREAD

**Discussion by Lysander086** [community/community#45312] [Copilot Conversations]
Displayed error: 
Sign in failed. Reason: Request signInInitiate failed with message: self signed certificate in certificate chain, request id: 3, error code: -3260

**Comment by jge162** [comment]
Did your certificate expire? The error message can happen if the certificate is not signed by a recognized certificate authority (CA) or if the certificate has expired.

You need to ensure that the certificate being used by GitHub is up to date and that it is signed by a recognized CA. Another solution is to install the self-signed certificate on your device so that it is recognized as a trusted certificate.

GL

**Comment by zmes50416** [comment]
You may have a proxy intercepting secure traffic through a self-signed cert, if you are using corporate devices.
I fix it by exporting corresponding cert and save it to NODE_EXTRA_CA_CERTS variable.

Here is the original instruction: https://sidd.io/2023/01/github-copilot-self-signed-cert-issue/

**Reply by Lysander086** [reply]
Finally see some lights on incapableness to use copilot on my computer. Great thx.

**Reply by vishanu1** [reply]
This solution didn't work for me :(
Any thoughts what else can be the problem ? 
Bhy the way are you using the latest 1.2.*** version of Copilot ?

**Comment by rodxotwod** [comment]
how do you know which certificate to export?

**Reply by zmes50416** [reply]
You can just visited github.com from your browser, and check the cert from the address bar. 
You probably will find the root cert different than the offical one (DigiCert). Find this cert in your OS native management tool and export it.
